src/librustdoc/html/static/themes/ayu.css

```diff
@@ -178,6 +178,9 @@ pre {
 .content span.externcrate, .content span.mod, .content a.mod {
 	color: #acccf9;
 }
+.content ul.crate a.crate {
+	font: 16px/1.6 "Fira Sans";
+}
```

Comment:
@camelid, @GuillaumeGomez why was this was added only to `ayu.css` and not `rustdoc.css` so it could apply to all themes?

Reply:
Hmm, I might have put that in the wrong spot by mistake.

Reply:
Yeah, it looks like the change didn't end up working. I was having issues with my build, so I wasn't able to test it locally.

Reply:
I'll see if I can get my build working and then open a new PR to fix that.

Reply:
Oh no, it's rebuilding LLVM...

Reply:
Fair enough. You should be able to avoid rebuilding rustc etc. by building docs for a single crate with `rustdoc -Z unstable-options --disable-minification --enable-index-page foo.rs` with a recent nightly then you can test css changes by just editing the generated files.

Reply:
Thanks! That's a good idea.

Reply:
Opened #76453.